Makefile

```diff
@@ -69,6 +69,19 @@ origin-tests-disconnected-environment:
 		ORIGIN_TESTS_REPOSITORY="$(ORIGIN_TESTS_REPOSITORY)" ORIGIN_TESTS_FILTER="$(ORIGIN_TESTS_FILTER)" \
 		hack/run-origin-tests.sh
 
+origin-tests-on-ci:
+	export SHARED_DIR=/tmp/shared
+	rm -rf /tmp/gcloudconfig/
+	export TEST_CSI_DRIVER_MANIFEST=""
+	export TEST_TYPE=""
+	ci_operator_source_file="/go/src/github.com/openshift/release/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh"
```

Comment:
I hate this implicit dependency
if they change something in `openshift/release` we'll break.
cant we have the ci lane to be multi-steps and add it there?
@fedepaol

Reply:
Not sure I got the comment right, but the problem now is that the "test" step is all or none.
Ideally, we would like to have a "prepare for origin tests" step and a "test" step that we override, but that's not what we have today